readme.md

```diff
@@ -30,10 +30,26 @@ choco install hyper
 
 ## Contribute
 
-1. Install the dependencies
-  * If you are running Linux, install `icnsutils`, `graphicsmagick`, `xz-utils`, `npm`, `rpm` and `yarn`
-  * If you are running Windows, install `yarn` (If you use the MSI installer you will need to install npm), and `windows-build-tools` with `yarn global add windows-build-tools`.
-  * Yarn installation instructions: https://yarnpkg.com/en/docs/install
+Regardless of the platform you are working on, you will need to have Yarn installed. If you have never installed Yarn before, you can find out how at: https://yarnpkg.com/en/docs/install.
+
+1. Install necessary packages:
+  * Windows
+    - `windows-build-tools`
+        + Install with `yarn global add windows-build-tools`.
+    - `npm` (If you use the MSI installer.)
```

Comment:
Do we still need  `npm`? 🤔

Reply:
I took a look into it and it seems unnecessary. The original issue didn't address this, but I can remove the references to `npm` if that helps.